services/dynamic-scheduler/tests/conftest.py

```diff
@@ -101,3 +126,13 @@ async def app(
         shutdown_timeout=None if is_pdb_enabled else MAX_TIME_FOR_APP_TO_SHUTDOWN,
     ):
         yield test_app
+
+
+@pytest.fixture
+async def remove_redis_data(redis_service: RedisSettings) -> None:
+    async with RedisClientsManager(
+        {RedisManagerDBConfig(x) for x in RedisDatabase}, redis_service
+    ) as manager:
+        await logged_gather(
+            *[manager.client(d).redis.flushall() for d in RedisDatabase]
```

Comment:
is this not a sign that things are not properly removed if you need to flush everything? also we do have fixtures for redis already, why is this one needed now?

Reply:
I disagree.
1. why are we not tracking them? can't you start to do so instead of flushing everything? (flushing everything means it is impossible to run tests in parallel + if fixtures are creating stuff randomly then they are bad). let's discuss.
2. I don't get what you say with servicelib. pytest-simcore can use anything.

Reply:
> I disagree.
> 
> 1. why are we not tracking them? can't you start to do so instead of flushing everything? (flushing everything means it is impossible to run tests in parallel + if fixtures are creating stuff randomly then they are bad). let's discuss.

I'm really unsure how to do this since it's the code that's being tested which generates these items.

> 2. I don't get what you say with servicelib. pytest-simcore can use anything.

pytest-simcore does not install servicelib that's why I'm saying this.